.changeset/gentle-ducks-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-testing": major
```

Comment:
Major because removing `isolateModules` is an API breaking change even though I don't know that anything uses it.